async/debounce.ts

```diff
@@ -36,8 +36,10 @@ export interface DebouncedFunction<T extends Array<unknown>> {
  * // output: Function debounced after 200ms with baz
  * ```
  *
+ * @typeParam T The arguments of the provided function.
```

Comment:
Nit: descriptions don't have to have their starting columns aligned.

Reply:
most parts of the codebase don't have that, only saw that in a single place. some even use `-` as a separator between argument and description.